relayer/chains/cosmos/query.go

```diff
@@ -160,14 +162,44 @@ func (cc *CosmosProvider) QueryBalanceWithAddress(ctx context.Context, address s
 	return res.Balances, nil
 }
 
+func (cc *CosmosProvider) QueryConsumerUnbondingPeriod(ctx context.Context) (time.Duration, error) {
```

Comment:
looks like this could be an unexported method since it's only ever called inside `QueryUnbondingPeriod`

Reply:
good call, done